index.js

```diff
@@ -6,6 +6,7 @@ import { promisify } from 'util';
 const exec = promisify(require('child_process').exec);
 
 let list;
+let dev='';
```

Comment:
can we rename dev to device

Reply:
Since we are already using device in another method I will be changing it to deviceParam @saikrishna321 pls. confirm

Reply:
sorry should be simParam

Reply:
simBootParam we are already using for getting all params, wouldn't it make confusing if similar name is used for deviceName param. @saikrishna321